cmd/auth/auth.go

```diff
@@ -11,10 +14,38 @@ var authCmd = &cobra.Command{
 	Short: "Authentication related commands",
 }
 
-var perisistentAuth auth.PersistentAuth
+var persistentAuth auth.PersistentAuth
+
+func promptForHost(ctx context.Context) error {
```

Comment:
I suggest to instead return (string, error) instead of setting a global variable in this function